README.md

```diff
@@ -168,6 +168,15 @@ EventsApi api = new ApiClient("API KEY", "API-SECRET")
                 .createService(EventsApi.class);
 ```
 
+By default, the Java SDK will upload to the [US1 Data Center URL](https://docs.mparticle.com/developers/server/http/#endpoint).  If the API key you're sending to exists in an mParticle Data Center that's not US1, you can add the following configuration to the SDK initialization:
+
+```java
+Retrofit.Builder mybuilder = new Retrofit.Builder().baseUrl("DATA CENTER URL").build();
```

Comment:
```suggestion
Retrofit.Builder mybuilder = new Retrofit.Builder().baseUrl("https://s2s.eu1.mparticle.com/v2/").build();
```

Reply:
if i were a developer i'd want to know if i need to start it with https, end it with a slash, should it contain `/events` etc - so better to include a real example

Reply:
smart!